src/main/java/teammates/common/util/Const.java

```diff
@@ -57,6 +57,20 @@
         public static final String QUEUE_XML_PATH = "src/main/webapp/WEB-INF/queue.xml";
         public static final String DEFAULT_PROFILE_PICTURE_PATH = "/images/profile_picture_default.png";
         
+        // The library files are chosen based on the following consideration:
+        // 1. Use local files on development to enable purely offline testing
+        // 2. Use CDN files on production to reduce the load on appspot's server
+
+        public static final String getjQueryFilePath(boolean isDevServer) {
```

Comment:
Perhaps give it as a javadoc comment for each method? That way it will show up in the IDE when someone tries to use the method. e.g. `Returns the ... when on dev env .. but returns ... on production ... to avoid the overhead ...`
Comments in between methods are not attached to method and can detached from methods during refactoring. 
Do the same for the other three methods in `Config.java`